sdcm/sct_config.py

```diff
@@ -499,6 +499,9 @@ class SCTConfiguration(dict):
              In case of random option - Scylla will start with different (random) shards on every node of the cluster
              """),
 
+        dict(name="nodes_smp", env="SCT_NODES_SMP", type=list,
+             help="List of shard numbers of nodes in Scylla cluster; list of int, like [4, 5, 3]"),
```

Comment:
```suggestion
             help="List of shard number to set per node in Scylla cluster; list of int, like [4, 5, 3]"),
```

I wonder how it would work with multi-dc cases:

```yaml
region_name: 'eu-west-1 us-east-1'
n_db_nodes: '2 1'
nodes_smp: [12, 12, 15]
```

Reply:
The number is based on node_index and I think it does not depend on dc